hoomd/logging.py

```diff
@@ -568,9 +568,9 @@ class they come from. For instance, the `hoomd.md.pair.LJ` class has a
     ``categories`` determines what if any types of loggable quantities (see
     `LoggerCategories`) are appropriate for a given `Logger` object. This helps
     logging backends determine if a `Logger` object is compatible. The
-    ``only_default`` flag is mainly a convenience by allowing quantities not
-    commonly logged (but available) to be passed over unless explicitly asked
-    for. You can override the ``only_default`` flag by explicitly listing the
+    ``only_default`` flag affects performance by controlling whether available
```

Comment:
@cbkerr `only_default` doesn't really effect performance. It is just to prevent qunatities that may not be commonly desired from automatically being added on `Logger.__iadd__` and `Logger.add`.